src/riak_kv_console.erl

```diff
@@ -1024,10 +1024,9 @@ bucket_type_and_table_error_local_key_test() ->
         "user   varchar   not null, ",
         "time   timestamp not null, ",
         "other  varchar   not null, ",
-        "PRIMARY KEY ((series, user, quantum(time, 15, m)), "
-        "series, user, time, other))">>,
+        "PRIMARY KEY ((series, user, quantum(time, 15, m)), seriesd, user, time, other))">>,
```

Comment:
Can you confirm this is now intentionally an error because of the ```seriesd``` field name rather than the additional ```other``` field, and if so, fix the name to reflect the missing field error.